src/rmrk2.0.0/tools/consolidator/interactions/mint.ts

```diff
@@ -23,6 +23,14 @@ export const validateMintNFT = async (
       `Attempted issue of NFT in non-owned collection. Issuer: ${nftParentCollection.issuer}, caller: ${remark.caller}`
     );
   }
+  
+  const nfts = await dbAdapter.getNFTsByCollection(nftParentCollection.id);
```

Comment:
Although I think this is good, I am a little bit worried on how this will affect consolidation time for large dumps like on RMRK2

Reply:
I used the way it was written in the destroy interaction as template not sure how it could be done differently

Reply:
Maybe a current nft number field for collections?

Reply:
Might be useful for getting the current nft number in the ui too